packages/web-fragments/src/elements/fragment-host.ts

```diff
@@ -70,7 +70,8 @@ export class FragmentHost extends HTMLElement {
 		this.neutralizeScriptTags();
 
 		// Preserve the existing stylesheets to avoid a FOUC when reinserting this element into the DOM
-		this.preserveStylesheets();
+		// Skipping due to CORS issues with square-sans.
+		// this.preserveStylesheets();
```

Comment:
our cdn font stylesheet fails when `cssRules` is read inside `preserveStylesheets` due to CORS issues. We need to change our style tag, still TODO but on the dashboard code side. not sure what reframed/fragments can do to handle this. maybe warn and no-op?